packages/alfa-rules/src/sia-r19/rule.ts

```diff
@@ -28,21 +32,22 @@ export default Rule.Atomic.of<Page, Attribute>({
           .filter(hasNamespace(Namespace.HTML, Namespace.SVG))
           .flatMap((element) =>
             Sequence.from(element.attributes).filter(
-              and(
-                property("name", aria.Attribute.isName),
-                property("value", not(isEmpty))
-              )
+              and(isAriaAttribute, property("value", not(isEmpty)))
```

Comment:
Using a `Refinement` allows to correctly narrow the targets to `Attribute<aria.Attribute.Name>`, which in turn let us get rid of that test in the Expectation.